lib/medtronic/processData.js

```diff
@@ -25,7 +25,7 @@ var common = require('../commonFunctions');
 var annotate = require('../eventAnnotations');
 var TZOUtil = require('../TimezoneOffsetUtil');
 
-var debug = (typeof __DEBUG__ === 'undefined') ? false : __DEBUG__;
+var debug = isBrowser ? require('../bows')('MedtronicDriver') : console.log;
```

Comment:
`isBrowser` isn't defined until the line below, so it will always be false here.